src/main/io/beeper.c

```diff
@@ -55,13 +55,18 @@ PG_REGISTER_WITH_RESET_TEMPLATE(beeperDevConfig_t, beeperDevConfig, PG_BEEPER_DE
 #endif
 #ifdef BEEPER
 #define BEEPER_PIN      BEEPER
+#define BEEPER_PWM_FREQ 0
+#elif defined(BEEPER_PWM)
+#define BEEPER_PIN      BEEPER_PWM
```

Comment:
Sorry, I misunderstood/overlooked that.
I would prefer to have BEEPER as general BEEPER enable (without value) and specify default pin in another variable, but it is already implemented this way, so keep it ..